Source/AtomicEditor/Editors/JSResourceEditor.cpp

```diff
@@ -202,6 +203,7 @@ bool JSResourceEditor::OnEvent(const TBWidgetEvent &ev)
             map["ForceSuperDown"] = true;
             #else
             map[KeyUp::P_QUALIFIERS] = QUAL_SHIFT | QUAL_CTRL;
+			webClient_->SendFocusEvent();
             #endif
```

Comment:
Also didn't help that the diff view in SourceTree was converting tabs to spaces so it didn't show up... both have now been changed.